datatypes/src/main/java/org/hyperledger/besu/datatypes/StorageSlotKey.java

```diff
@@ -31,7 +32,7 @@
  */
 public class StorageSlotKey implements Comparable<StorageSlotKey> {
 
-  private final Hash slotHash;
+  private final Supplier<Hash> slotHash;
```

Comment:
Suggest MemoizingSupplier here since the slotKey is final

Reply:
yeah forgot thanks